Sources/MapboxMaps/Foundation/CameraManagerProtocol.swift

```diff
@@ -120,15 +120,14 @@ internal protocol CameraManagerProtocol {
 
     // MARK: - Camera getters/setters
 
-    /// :nodoc:
     /// Changes the map view by any combination of center, zoom, bearing, and pitch,
     /// without an animated transition. The map will retain its current values
     /// for any details not passed via the camera options argument. It is not
     /// guaranteed that the provided `CameraOptions` will be set, the map may apply
     /// constraints resulting in a different `CameraState`.
     ///
     /// - Parameter cameraOptions: New camera options
```

Comment:
Let's add an `- Important:` section about needing to call `cancelAnimations` (as discussed offline).

Reply:
+1